storage/innobase/include/trx0purge.h

```diff
@@ -438,6 +396,11 @@ class purge_sys_t
   @param	already_stopped	True indicates purge threads were
 				already stopped */
   void stop_FTS(const dict_table_t &table, bool already_stopped=false);
+
+  /** Cleanse purge queue to remove the rseg that reside in undo-tablespace
+  marked for truncate.
+  @param[in] space undo tablespace being truncated */
+  void cleanse_purge_queue(const fil_space_t& space);
```

Comment:
Let’s use plain `@param` in new code. A reference to `const` can only be an input parameter.